src/app/core/data/request.service.ts

```diff
@@ -145,14 +145,11 @@ export class RequestService {
    * Configure a certain request
    * Used to make sure a request is in the cache
    * @param {RestRequest} request The request to send out
-   * @param {boolean} forceBypassCache When true, a new request is always dispatched
    */
-  configure<T extends CacheableObject>(request: RestRequest, forceBypassCache: boolean = false): void {
+  configure<T extends CacheableObject>(request: RestRequest): void {
     const isGetRequest = request.method === RestRequestMethod.GET;
-    if (forceBypassCache) {
-      this.clearRequestsOnTheirWayToTheStore(request);
-    }
-    if (!isGetRequest || (forceBypassCache && !this.isPending(request)) || !this.isCachedOrPending(request)) {
+    const isSubmission =  request instanceof SubmissionRequest;
```

Comment:
This might not be the right solution, but it does appear to fix the problem noticed by @atarix83 

I think previously the combination of forceBypassCache and removing pending requests from `requestsOnTheirWayToTheStore` had the effect of forcing a new dispatch request. Without this, a lookup by href is attempted and fails. Checking for the request type is a different way to get the same result but may have unintended consequences.  

Also, I noticed that `SubmissionService` and `RemoteDataBuildService` are both importing methods from the `shared/testing/utils` It seems like those methods would be better located in a different utility.